spec/associations_spec.cr

```diff
@@ -1,5 +1,11 @@
 require "./spec_helper"
 
+# class CredentialedUser < User
+#   table users do
+#     has_one sign_in_credential : SignInCredential
+#   end
+# end
```

Comment:
Do we need this model anymore or can it be removed?

Reply:
I forgot about that one, its removed now